src/utils/Fetcher.ts

```diff
@@ -39,11 +50,11 @@ export class Fetcher {
    * @param writeTo the stream to pipe the response to
    * @param options config for the request
    */
-  fetchPipe(url: string, writeTo: ReadableStream<Uint8Array>, options?: RequestOptions): Promise<Headers> {
+  fetchPipe(url: string, writeTo: any, options?: RequestOptions): Promise<Headers> {
     return fetchPipe(url, writeTo, mergeRequestOptions(this.customDefaults, options))
   }
 
-  postForm(url: string, options?: RequestOptions): Promise<any> {
+  postForm(url: string, options?: RequestOptions): Promise<unknown> {
```

Comment:
TS Rule of thumb: When you don't know the type, use `unknown` when you don't care, use `any`